charts/fluent-bit/values.yaml

```diff
@@ -9,8 +9,9 @@ replicaCount: 1
 image:
   repository: cr.fluentbit.io/fluent/fluent-bit
   # Overrides the image tag whose default is {{ .Chart.AppVersion }}
-  tag: ""  # Set to "-" to not use the default value
-  digest: ""
+  # Set to "-" to not use the default value
+  tag:
+  digest:
```

Comment:
Nice, good.

Can you add a comment to indicate default behaviour is to not check digest if not defined?

Reply:
@stevehipwell do we actually use the digest in the helper?

Reply:
Digest is used, the docs on this chart are poor and I'm focusing my effort of the separate charts so maybe someone else could look at implementing `helm-docs` on the existing chats and getting a good README?